### DISCUSSION TITLE
post/building-a-forum-with-react-nodejs-4sm2wj

### DISCUSSION THREAD

**Discussion by giscus** [alexandre-dev29/community-blog#3] [Articles]
# post/building-a-forum-with-react-nodejs-4sm2wj

In this article, you'll learn how to build a forum system that allows users to create, react, and reply to post threads. In the end, we will also send a notification on each reply on a thread with Novu, you can skip the last step if you want only the technical.

https://blogcommunity.axelmwenze.dev/post/building-a-forum-with-react-nodejs-4sm2wj

**Comment by alexandre-dev29** [comment]
great work

**Reply by lyabs243** [reply]
Great :)

**Comment by lyabs243** [comment]
Hey !!!
